### PR TITLE
chore: remove redundant code

### PR DESCRIPTION
**Pull request by PritishBudhiraja**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by jainlokesh318 — APPROVED** [review]

File: src/components/SelectBox.resi [hunk 1]
```diff
@@ -106,13 +196,13 @@ module BaseRadio: {
...
     ~fill: string=?,
-    ~customStyle: Js.String2.t=?,
+    ~customStyle: string=?,
     ~searchable: bool=?,
     ~isMobileView: bool=?,
     ~customSearchStyle: string=?,
     ~descriptionOnHover: bool=?,
     ~addDynamicValue: bool=?,
-    ~dropdownCustomWidth: Js.String2.t=?,
+    ~dropdownCustomWidth: string=?,
```
Comment (jainlokesh318) [inline src/components/SelectBox.resi:205]: These changes in SelectBox types can be moved to seperate PR?

File: src/context/ThemeProvider.res [hunk 2]
```diff
@@ -18,7 +18,6 @@ let useTheme = () => {
 }
 
 @val external window: Js.Nullable.t<'a> = "window"
```
Comment (jainlokesh318) [inline src/context/ThemeProvider.res:20]: can this be removed?

File: src/components/SelectBox.resi (hunk 1 above)
Comment (PritishBudhiraja) [reply src/components/SelectBox.resi:205]: Because of Window changes these have come. So it should be in this PR.

**Review by jainlokesh318 — APPROVED** [review]

File: src/context/ThemeProvider.res (hunk 2 above)
Comment (PritishBudhiraja) [reply src/context/ThemeProvider.res:20]: Updated

**Review by jainlokesh318 — APPROVED** [review]

**Review by jainlokesh318 — APPROVED** [review]

**Review by sagarnaikjuspay — APPROVED** [review]

**Pull request merged**